berkeley-mobile/Calendar/Academic/AcademicCalendarViewController.swift

```diff
@@ -134,70 +54,24 @@ extension AcademicCalendarViewController {
         scrollingStackView.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
         scrollingStackView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
     }
-
-    // MARK: Upcoming Card
-
-    private func setupUpcoming() {
-        let card = CardView()
-        card.layoutMargins = kCardPadding
-        scrollingStackView.stackView.addArrangedSubview(card)
-
-        let contentView = UIView()
-        contentView.layer.masksToBounds = true
-        card.addSubview(contentView)
-        contentView.translatesAutoresizingMaskIntoConstraints = false
-        contentView.setConstraintsToView(top: card, bottom: card, left: card, right: card)
-
-        let headerLabel = UILabel()
-        headerLabel.font = Font.bold(24)
-        headerLabel.text = "Upcoming"
-        contentView.addSubview(headerLabel)
-        headerLabel.translatesAutoresizingMaskIntoConstraints = false
-        headerLabel.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
-        headerLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
-        headerLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
-
-        let collectionView = CardCollectionView(frame: .zero)
-        collectionView.delegate = self
-        collectionView.dataSource = self
-        collectionView.contentInset = UIEdgeInsets(top: 0, left: card.layoutMargins.left, bottom: 0, right: card.layoutMargins.right)
-        contentView.addSubview(collectionView)
-        collectionView.translatesAutoresizingMaskIntoConstraints = false
-        collectionView.topAnchor.constraint(equalTo: headerLabel.bottomAnchor, constant: 16).isActive = true
-        collectionView.heightAnchor.constraint(equalToConstant: CardCollectionViewCell.kCardSize.height).isActive = true
-        collectionView.leftAnchor.constraint(equalTo: card.leftAnchor).isActive = true
-        collectionView.rightAnchor.constraint(equalTo: card.rightAnchor).isActive = true
-        collectionView.bottomAnchor.constraint(equalTo: card.layoutMarginsGuide.bottomAnchor).isActive = true
-
-        eventsCollection = collectionView
-        upcomingMissingView = MissingDataView(parentView: collectionView, text: "No upcoming events")
-    }
-
-    // MARK: Calendar Table
-
-    private func setupCalendarList() {
+    
+    // MARK: Calendar
+    private func setUpCalendar() {
         let card = CardView()
         card.layoutMargins = kCardPadding
-        scrollingStackView.stackView.addArrangedSubview(card)
+        scrollingStackView.addSubview(card)
```

Comment:
changed back to addArrangedSubview. that's probably better for consistency/in case we need to add other elements to the stack in the future. this fixed the alignment/margin issue